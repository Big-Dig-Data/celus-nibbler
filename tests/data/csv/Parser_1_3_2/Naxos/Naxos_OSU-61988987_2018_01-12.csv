Subscriber Name:,Ostravska univerzita,,,,,,,,
,Jan-2018,Feb-2018,Mar-2018,Apr-2018,May-2018,Jun-2018,Jul-2018,Aug-2018,Sep-2018,Oct-2018,Nov-2018,Dec-2018
Tracks,4,0,0,0,0,0,0,0,0,3,0,7
