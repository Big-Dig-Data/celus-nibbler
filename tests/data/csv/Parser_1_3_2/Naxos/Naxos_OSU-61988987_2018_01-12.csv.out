"2018-01-01","2018-01-31","Naxos",,"Tracks",,,"4"
"2018-02-01","2018-02-28","Naxos",,"Tracks",,,"0"
"2018-03-01","2018-03-31","Naxos",,"Tracks",,,"0"
"2018-04-01","2018-04-30","Naxos",,"Tracks",,,"0"
"2018-05-01","2018-05-31","Naxos",,"Tracks",,,"0"
"2018-06-01","2018-06-30","Naxos",,"Tracks",,,"0"
"2018-07-01","2018-07-31","Naxos",,"Tracks",,,"0"
"2018-08-01","2018-08-31","Naxos",,"Tracks",,,"0"
"2018-09-01","2018-09-30","Naxos",,"Tracks",,,"0"
"2018-10-01","2018-10-31","Naxos",,"Tracks",,,"3"
"2018-11-01","2018-11-30","Naxos",,"Tracks",,,"0"
"2018-12-01","2018-12-31","Naxos",,"Tracks",,,"7"
